src/javascript/app/pages/cashier/cashier.js

```diff
@@ -216,7 +214,6 @@ const Cashier = (() => {
         if (Client.isLoggedIn()) {
             BinarySocket.send({ statement: 1, limit: 1 });
             BinarySocket.wait('authorize', 'mt5_login_list', 'statement', 'get_account_status', 'landing_company').then(() => {
-                checkStatusIsLocked(State.getResponse('get_account_status'));
```

Comment:
why do we need to remove this?

Reply:
back to previous version, thanks. No need to do it

Reply:
@farrah-deriv

Reply:
in progress, need to discuss this point

Reply:
Is this resolved?

Reply:
waiting reply in a channel #task_cashier_onboarding for this issue

Reply:
is this resolved?